icechunk-python/pyproject.toml

```diff
@@ -16,7 +16,7 @@ classifiers = [
 license = { text = "Apache-2.0" }
 dynamic = ["version"]
 
-dependencies = ["zarr==3.0.0rc2"]
+dependencies = ["zarr>=3"]
```

Comment:
Is this the right way to specify it? should it be 3 but not for?

Reply:
oh you want `"zarr>=3<4"`?

cc @jhamman for input